Makefile

```diff
@@ -1,6 +1,8 @@
 branch = $(shell git rev-parse --abbrev-ref HEAD)
+export SSL_CERT_FILE = /etc/ssl/certs/ca-certificates.crt
+export CURL_CA_BUNDLE = /etc/ssl/certs/ca-certificates.crt
```

Comment:
This actually fies that @zhenik 
Maybe your certs are out of date. Run `aurora-ske-certificates && sudo update-ca-certificates`

Reply:
Yes. It works. 
1. Would you like to add this commands to Makefile `aurora-ske-certificates && sudo update-ca-certificates` ?
2. You could also remove whitespace in exports 

![39](https://user-images.githubusercontent.com/15572799/84023357-3e0e6a80-a988-11ea-9c37-e015c8cd7434.PNG)

Reply:
![image](https://user-images.githubusercontent.com/51820995/84023320-2636e680-a988-11ea-82ee-20368bb6f472.png)
works for me.

Reply:
it works for me as well, after I ran `aurora-ske-certificates && sudo update-ca-certificates`

Reply:
I only ran `sudo update-ca-certificates`.

Reply:
I am not going to add those, as they are never needed again. (root-ca trust expired last week and a reboot should fix this.)